nautobot/tenancy/api/serializers.py

```diff
@@ -45,19 +32,4 @@ class TenantSerializer(NautobotModelSerializer, TaggedModelSerializerMixin):
 
     class Meta:
         model = Tenant
-        fields = [
-            "url",
-            "name",
-            "tenant_group",
-            "description",
-            "comments",
-            "circuit_count",
-            "device_count",
-            "ipaddress_count",
-            "prefix_count",
-            "rack_count",
-            "virtualmachine_count",
-            "vlan_count",
-            "vrf_count",
-            "cluster_count",
```

Comment:
Have we confirmed manually that the `count` fields get included/populated correctly? I don't have confidence in the thoroughness of our unit tests to test this.

Reply:
https://github.com/nautobot/nautobot/issues/3609